docs/content/manpages/geowave-localToGW.txt

```diff
@@ -0,0 +1,39 @@
+//:geowave-localToGW(1)
+//:======================
+//::doctype: manpage
+
+NAME
+//:----
+
+geowave ingest localToGW - Ingest supported files in local file system directly, without using HDFS
+
+SYNOPSIS
+//:--------
+
+*geowave ingest localToGW* [options] <file or directory> <storename> <comma delimited index/group list>
+
+DESCRIPTION
+//:-----------
+
+The geowave ingest localToGW operator will run the ingest code (parse to features, load features to geowave) against local file system content.
+
+OPTIONS
+//:-------
+
+-x, --extension::
+
+   individual or comma-delimited set of file extensions to accept (optional)
+
+-f, --formats::
```

Comment:
So each of the formats has it's own set of options... and when you specify them, they'll show up, like:

"geowave ingest localtogw -f geotools-vector,gpx"